java/configuration/src/main/java/sleeper/configuration/properties/instance/InstanceProperty.java

```diff
@@ -35,6 +35,8 @@ static Optional<InstanceProperty> getByName(String propertyName) {
         return Index.INSTANCE.getByName(propertyName);
     }
 
+    SleeperProperty getDefaultProperty();
```

Comment:
It doesn't look like this is ever used? Please could you add some unit tests for the behaviour of default properties in InstanceProperties?

Reply:
Addressed